kmath-functions/src/commonMain/kotlin/space/kscience/kmath/functions/LabeledPolynomial.kt

```diff
@@ -0,0 +1,619 @@
+/*
+ * Copyright 2018-2021 KMath contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+@file:Suppress("NOTHING_TO_INLINE", "KotlinRedundantDiagnosticSuppress")
+
+package space.kscience.kmath.functions
+
+import space.kscience.kmath.expressions.Symbol
+import space.kscience.kmath.operations.Ring
+import kotlin.jvm.JvmName
+import kotlin.math.max
+
+
+/**
+ * Represents multivariate polynomial that stores its coefficients in a [Map] and terms' signatures in a [Map] that
+ * associates variables (of type [Symbol]) with their degree.
+ *
+ * @param C the type of constants.
+ */
+public data class LabeledPolynomial<C>
+@PublishedApi
```

Comment:
I don't understand what this thing does. Could you provide a reference to its description and usage?

Reply:
Sorry, are you asking about [`@PublishedAPI`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-published-api/)?